### DISCUSSION TITLE
Reload/Refresh/Re-render the div content (customize block)

### DISCUSSION THREAD

**Discussion by censonalvin** [GrapesJS/grapesjs#4916] [Q&A]
Is there a way to reload/refresh a div content (customize block) after clicking a button ? Like when i click a refresh button to refresh/reload/re render the added content for the customize block (mainly image).

I do try the (#id).load() of jquery in the ajax success but the div content not re appearing again.

**Comment by artf** [comment]
Are you talking about rerendering a specific component in the canvas?

**Reply by censonalvin** [reply]
not on the canvas but the div that the Block/BlockManager was append to. Is there a way to re render it without reloading the page?

**Reply by artf** [reply]
Can you elaborate why would you need to rerender appended block? Are you using some custom component? Components appended in the canvas are expected to rerender automatically when it's necessary.
